### ISSUE TITLE
Encrypting your Model Attributes

### ISSUE THREAD

**Issue by bvipul** [laravel/blog-contest-may-mayhem#84]
https://codeburst.io/encrypting-your-model-attributes-2116bf623d10

**Comment by shirshak55** [comment]
Hi nice package. But If our key gets lost will all data gets lost ?

**Comment by bvipul** [comment]
Yes, I am afraid, that would be the case.

**Comment by koenhoeijmakers** [comment]
Would be nice if you could tell the trait which attributes to encrypt :)

**Comment by bvipul** [comment]
Yes, currently I am trying to encrypt the fillables, but I am thinking to change that to a model variable. Thanks for suggestion, and I also need to encrypt the binding in builder while querying...will do these in the next build
